.gcloudignore

```diff
@@ -0,0 +1,18 @@
+.gcloudignore
```

Comment:
this file is to be explicit in what we do not want to upload during our App Engine deploys